src/masternodes/mn_rpc.cpp

```diff
@@ -1558,32 +2005,675 @@ UniValue accounttoutxos(const JSONRPCRequest& request) {
+               HelpRequiringPassphrase(pwallet) + "\n",
                {
+                   {"metadata", RPCArg::Type::OBJ, RPCArg::Optional::OMITTED, "",
+                       {
+                            {"tokenA", RPCArg::Type::STR, RPCArg::Optional::NO,
+                            "One of the keys may be specified (id/symbol)"},
+                            {"tokenB", RPCArg::Type::STR, RPCArg::Optional::NO,
+                            "One of the keys may be specified (id/symbol)"},
+                            {"commission", RPCArg::Type::NUM, RPCArg::Optional::NO,
+                            "Pool commission, up to 10^-8"},
+                            {"status", RPCArg::Type::BOOL, RPCArg::Optional::NO,
+                            "Pool Status: True is Active, False is Restricted"},
+                            {"ownerAddress", RPCArg::Type::STR, RPCArg::Optional::NO,
+                            "Address of the pool owner."},
+                            {"pairSymbol", RPCArg::Type::STR, RPCArg::Optional::OMITTED,
+                             "Pair symbol (unique), no longer than " +
+                             std::to_string(CToken::MAX_TOKEN_SYMBOL_LENGTH)},
+                       },
+                   },
+                   {"inputs", RPCArg::Type::ARR, RPCArg::Optional::OMITTED_NAMED_ARG, "A json array of json objects",
+                       {
+                           {"", RPCArg::Type::OBJ, RPCArg::Optional::OMITTED, "",
+                               {
+                                   {"txid", RPCArg::Type::STR_HEX, RPCArg::Optional::NO, "The transaction id"},
+                                   {"vout", RPCArg::Type::NUM, RPCArg::Optional::NO, "The output number"},
+                               },
+                           },
+                       },
+                   },
                },
                RPCResult{
-                       "{balance_type:value,...}     (array) Json object with accounts information\n"
+                       "\"hash\"                  (string) The hex-encoded hash of broadcasted transaction\n"
                },
                RPCExamples{
-                       HelpExampleCli("listcommunitybalances", "")
-                       + HelpExampleRpc("listcommunitybalances", "")
+                       HelpExampleCli("createpoolpair",   "\"{\\\"tokenA\\\":\\\"MyToken1\\\","
+                                                          "\\\"tokenB\\\":\\\"MyToken2\\\","
+                                                          "\\\"commission\\\":\\\"0.001\\\","
+                                                          "\\\"status\\\":\\\"True\\\","
+                                                          "\\\"ownerAddress\\\":\\\"Address\\\""
+                                                          "}\" \"[{\\\"txid\\\":\\\"id\\\",\\\"vout\\\":0}]\" ")
+                       + HelpExampleRpc("createpoolpair", "\"{\\\"tokenA\\\":\\\"MyToken1\\\","
```

Comment:
These two examples are the same, should keep only one or update one for them.

Reply:
Not exectly. One for "cli", one for "rpc" (HelpExampleCli, HelpExampleRpc) - it is a "common" practice in all bitcoin's rpcs.

Reply:
I can just remove one, if you want

Reply:
@DeFiDev that's ok, can keep both if they are for CLI and RPC